Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
I agree with both of you, and as you can see from my original comment I didn't have a strong opinion either way.

For consistency with other punctuation operators, including the `::` in PHP, let's go with `punctuation.accessor`. That was my first instinct, however the docs I had been reading about it made it sound as if it was doing more. I think the visual consistency and style-ability wins here over pure semantic correctness. Kind of like how we scope docstrings in Python as comments because they are used as comments, but technically are just strings.